src/app/Writers/TableWriter.php

```diff
@@ -59,7 +59,7 @@ private function templateFromTo()
 
         $array = [
             '${permissionGroup}' => $this->choices->get('permissionGroup')->get('name'),
-            '${Models}'          => Str::plural($model),
+            '${Models}'          => Str::plural(Str::snake($model)),
```

Comment:
Apparently this is not used anywhere in the stub. Am I wrong?

We could use `Models` for determining the table's `name` attribute

Reply:
It is:
- https://github.com/laravel-enso/StructureManager/blob/214b40bfb1c875751de546a2ad2a378dfda77810/src/app/Writers/stubs/table/template.stub#L2
- https://github.com/laravel-enso/StructureManager/blob/214b40bfb1c875751de546a2ad2a378dfda77810/src/app/Writers/TableWriter.php#L62

Reply:
Lol. I added this yesterday, before it was not used

Reply:
I just did a quick search 😂 
It was used somewhere though... it's how I tracked it down

Reply:
Just used my local files to search; it was actually the lowercase version, my bad.

![image](https://user-images.githubusercontent.com/932193/56884842-4305a880-6a6b-11e9-8224-ac37bfacd59d.png)

I believe this is still incorrect though.

Reply:
why?

Reply:
Unless you changed some related code this is generating camel instead of snakecase, but tables are snake-cased in Laravel

Reply:
ah you're right